From cafc0ce510765ca3aaaf028599d1498193951d2e Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Mon, 02 Nov 2020 17:50:37 +0200
Subject: [PATCH] [SW-12754] gaudi: security: remove todo

The TODO ragarding the SRAM_Y_X_RTRs has already been resolved.
Remove the TODO, but keep the comment for information.
No functional changes.

Change-Id: Ibd522634e014ecfa5f96a266e4489f8cf9843390
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/gaudi/security.c b/gaudi/security.c
index 5a769dc..c8ea1e6 100644
--- a/gaudi/security.c
+++ b/gaudi/security.c
@@ -1038,11 +1038,11 @@
 	set_prot_bits_pages_range_all(base, SRAM_BANKS_DIST, SRAM_BANKS_NUM);
 
 	/*
-	 * TODO: There is an integration version mismatch bug in Gaudi
+	 * There is an integration version mismatch bug in Gaudi
 	 * resulting in absence of the Privilege protection bits registers.
 	 * Therefore we cannot use the protection bits to protect the
 	 * SRAM_Y_X_RTRs.
-	 * LBW RRs should be used instead.
+	 * LBW RRs are used instead.
 	 */
 }
 
